apps/examples/rt_ocf_test/rt_ocf_test_main.c

```diff
@@ -0,0 +1,738 @@
+			if (queries->if_query && strcmp(OIC_IF_BASELINE_VALUE, queries->if_query) == 0) {
+				ocf_res_add_if_rt_rep(rep, item->resource);
+			}
+			ocf_response_send(request, rep, OCF_RESPONSE_CHANGED);
+			ocf_rep_encoder_release(rep);
+
+			notifiy_item_state(&blue_led);
+			return;
+		}
+	}
+	ocf_response_send(request, NULL, OCF_RESPONSE_NOT_ACCEPTABLE);
+}
+
+static void red_led_post_handler_func(ocf_request_s request, ocf_rep_decoder_s data)
+{
+	printf("Red LED receives POST Method\n");
+	ocf_item *item = &red_led;
+
+	if (!request) {
+		printf("[%s]request is null\n", __func__);
+		return;
+	}
+
+	bool value = 0;
+	if (OCF_OK == ocf_rep_get_bool_from_map(data, "value", &value)) {
+		printf("[%s]value : %d\n", __func__, value);
+		item->value = value;
+
+		ocf_rep_encoder_s rep = create_item_representation(item);
+		if (rep) {
+			ocf_request_query_set_s *queries = ocf_request_get_queries(request);
+			if (queries->if_query && strcmp(OIC_IF_BASELINE_VALUE, queries->if_query) == 0) {
+				ocf_res_add_if_rt_rep(rep, item->resource);
+			}
+			ocf_response_send(request, rep, OCF_RESPONSE_CHANGED);
+			ocf_rep_encoder_release(rep);
+
+			notifiy_item_state(&red_led);
+			return;
+		}
+	}
+	ocf_response_send(request, NULL, OCF_RESPONSE_NOT_ACCEPTABLE);
+}
+
+static void delete_handler_func(ocf_request_s request, ocf_rep_decoder_s data)
+{
+	printf("Delete request is received\n");
+	ocf_response_send(request, NULL, OCF_RESPONSE_METHOD_NOT_ALLOWED);
+	return;
+}
+
+static void create_ocf_item_resource(ocf_item *item)
+{
+	item->resource = ocf_res_new_resource(item->uri);
+	ocf_res_set_discoverable(item->resource, true);
+	ocf_res_set_observable(item->resource, true);
+	ocf_res_set_interface(item->resource, OIC_IF_BASELINE);
+	ocf_res_set_default_interface(item->resource, OIC_IF_BASELINE);
+	const char *item_resource_types[1] = { "core.light" };
+	ocf_res_set_resource_types(item->resource, item_resource_types, 1);
+	ocf_res_set_secure(item->resource, true);
+	ocf_res_set_resource_protocol(item->resource, OCF_COAP | OCF_COAPS);
+	ocf_res_set_request_handler(item->resource, OCF_GET, item->get);
+	ocf_res_set_request_handler(item->resource, OCF_PUT, item->put);
+	ocf_res_set_request_handler(item->resource, OCF_POST, item->post);
+	ocf_res_set_request_handler(item->resource, OCF_DELETE, item->delete);
+	ocf_res_register_resource(item->resource);
+}
+
+/* Set PS handler */
+static ocf_persistent_storage_handler_s temp_doxm_handler = { test_doxm_fopen, fread, fwrite, fclose };
+static ocf_persistent_storage_handler_s temp_cred_handler = { test_cred_fopen, fread, fwrite, fclose };
+static ocf_persistent_storage_handler_s temp_acl2_handler = { test_acl2_fopen, fread, fwrite, fclose };
+static ocf_persistent_storage_handler_s temp_pstat_handler = { test_pstat_fopen, fread, fwrite, fclose };
+
+static void *gpio_thread(void *data)
+{
+
+	int r_led = 45;
+	int b_led = 49;
+	int r_btn = 42;
+	int b_btn = 44;
+	int r_read, b_read;
```

Comment:
Let's define ```b_read``` in next line. That is our recommandation of coding rule.

Reply:
Modified.